src/HuC1.md

```diff
@@ -1,50 +1,51 @@
 # HuC1
 
-HuC1 is an MBC used by some Game Boy games which besides doing the usual
-MBC stuff, also provides IR comms. A lot of sources on the internet said
-that HuC1 was \"similar to MBC1\", but they didn\'t provide any detail.
-I took a look, and it turns out that HuC1 differs from MBC1 quite a lot.
+HuC1 is an MBC used by some Game Boy games which, besides doing the
+usual MBC stuff, also provides infrared communication. A lot of sources
+on the internet said that HuC1 was \"similar to MBC1\", but they didn\'t
+provide any detail.  I took a look, and it turns out that HuC1 differs
+from MBC1 quite a lot.
 
 ## Memory Map
 
 Address range | Feature
 --------------|------------------------------------
-  $0000-1FFF  | RAM/IR select (when writing only)
-  $2000-3FFF  | ROM bank select (when writing only)
-  $4000-5FFF  | RAM bank select (when writing only)
-  $6000-7FFF  | Nothing?
-  $A000-BFFF  | Cart RAM or IR register
+  $0000–1FFF  | RAM/IR select (when writing only)
+  $2000–3FFF  | ROM bank select (when writing only)
+  $4000–5FFF  | RAM bank select (when writing only)
+  $6000–7FFF  | Nothing?
+  $A000–BFFF  | Cart RAM or IR register
 
-### 0000-1FFF IR Select (Write Only)
+### 0000–1FFF IR Select (Write-Only)
 
 Most MBCs let you disable the cartridge RAM to prevent accidental
 writes. HuC1 doesn\'t do this, instead you use this register to switch
-the A000-BFFF region between \"RAM mode\" and \"IR mode\" (described
-below). Write 0x0E to switch to IR mode, and anything else to switch to
-RAM mode. Nevertheless some HuC1 games attempt to write 0x0A and 0x00 to
+the A000–BFFF region between \"RAM mode\" and \"IR mode\" (described
+below). Write $0E to switch to IR mode, and anything else to switch to
+RAM mode. Nevertheless some HuC1 games attempt to write $0A and $00 to
 this region as if it would enable/disable cart RAM.
 
-### 2000-3FFF ROM Bank Number (Write Only)
+### 2000–3FFF ROM Bank Number (Write-Only)
 
 HuC1 can accept a bank number of at least 6 bits here.
 
-### 4000-5FFF RAM Bank Select (Write Only)
+### 4000–5FFF RAM Bank Select (Write-Only)
 
 HuC1 can accept a bank number of at least 2 bits here.
 
-### 6000-7FFF Nothing? (Write Only)
+### 6000–7FFF Nothing? (Write-Only)
 
 Writes to this region seem to have no effect. Even so, some games do
 write to this region, as if it had the same effect as on MBC1. You may
 safely ignore these writes.
 
-### A000-BFFF Cart RAM or IR register (Read/Write)
+### A000–BFFF Cart RAM or IR register (Read/Write)
 
-When in \"IR mode\" (wrote 0x0E to 0x0000), the IR register is visible
-here. Write to this region to control the IR transmitter. 0x01 turns it
-on, 0x00 turns it off. Read from this region to see either 0xC1 (saw
-light) or 0xC0 (did not see light). When in \"RAM mode\" (wrote
-something other than 0x0E to 0x000) this region behaves like normal cart
+When in \"IR mode\" (wrote $0E to 0000), the IR register is visible
```

Comment:
```suggestion
When in \"IR mode\" (wrote $0E to $0000), the IR register is visible
```

Reply:
Well, I took the following from CONTRIBUTING.md to heart:

> To prevent clutter, don't use a prefix for hex numbers when it's clear from the context that a number is hexadecimal. For example, addresses and lists of opcodes. In those cases, zero-pad, even for smaller numbers: `0000-3FFF` instead of `0-3FFF`.

I padded addresses to four hexadecimal digits and didn’t add a prefix, as addresses are implied to be hexadecimal.  Should this rule be updated/clarified?

Reply:
Fair that you're following the rules; I think that addresses in running text like this should get a hex prefix at least for consistency ("0E" is arguably unambiguous), and the prefix helps it stand out from other words.